### PR TITLE
doc(readme): clean up readme and move some stuff to docs dir

### LINKED ISSUE #154: re work readme
our readme right now is a little confusing- it's got information for both contributors and end users and is generally a bit too much of a grab bag to be useful. i'd like to rework much of the content into the `/docs` director and use the `README` as a table of contents of sorts- i'll likely make a PR to kick this off today but am still super open to everyone's thoughts!

### PR DESCRIPTION
**Pull request by ashleygwilliams**
fixes #154

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,8 +42,15 @@ customize the log verbosity using the verbosity flag.
 | -vv           | All Debug, Info, Warn, and Errors are logged        |
 | -vvv          | All Trace, Debug, Info, Warn, and Errors are logged |
 
+## 👯 contributing
+
+Read our [guide] on getting up and running for developing `wasm-pack`, and
+check out our [contribution policy].
+
+[guide]: doc/contributing.md
+[contirbution policy]: contributing.md
```
Comment (mgattozzi) [inline README.md]: contirbution -> contribution
contributing.md -> CONTRIBUTING.md

File: README.md [hunk 2]
```diff
@@ -18,39 +22,16 @@ visiting that repo!
...
-  - optionally pass a scope name to generate a `package.json` for a scoped pkg, e.g.:
-    ```
-    wasm-pack init examples/scopes-hello-world --scope test
-    ```
-    generates a `package.json` for an npm package called `@test/scopes-hello-world`
-- 🍱  `pack`: create a tarball but don't push to the npm registry (see https://docs.npmjs.com/cli/pack)
-- 🎆  `publish`: create a tarball and publish to the npm registry (see https://docs.npmjs.com/cli/publish)
-
-### logging
+### 📝 logging
```
Comment (mgattozzi) [inline README.md]: Might as well just break this out into it's own separate file now that it's already done for the commands

File: README.md [hunk 3]
```diff
@@ -1,13 +1,17 @@
...
 [![Build Status](https://travis-ci.org/ashleygwilliams/wasm-pack.svg?branch=master)](https://travis-ci.org/ashleygwilliams/wasm-pack)
 [![Build status](https://ci.appveyor.com/api/projects/status/7jjuo5wewu9lyyfi?svg=true)](https://ci.appveyor.com/project/ashleygwilliams/wasm-pack)
 
-the goal of this project is to create a portable command line tool
-for publishing compiled wasm projects to the npm registry for the consumption
-of js devs using the npm CLI, yarn, or any other CLI tool that interfaces
-with the npm registry.
+this tool seeks to be a one-stop shop for building and working with rust-
+generated webassembly that you would like to interop with JavaScript, in the
+browser or with Node.js. `wasm-pack` helps you build and publish rust-generated
```
Comment (mgattozzi) [inline README.md]: rust-generated -> rust generated

File: README.md [hunk 4]
```diff
@@ -1,13 +1,17 @@
...
+web assembly to the npm registry to be used alongside any other javascript
```
Comment (mgattozzi) [inline README.md]: web assembly -> WebAssembly or wasm

File: README.md [hunk 5]
```diff
@@ -1,13 +1,17 @@
...
 
 [![Build Status](https://travis-ci.org/ashleygwilliams/wasm-pack.svg?branch=master)](https://travis-ci.org/ashleygwilliams/wasm-pack)
 [![Build status](https://ci.appveyor.com/api/projects/status/7jjuo5wewu9lyyfi?svg=true)](https://ci.appveyor.com/project/ashleygwilliams/wasm-pack)
 
-the goal of this project is to create a portable command line tool
-for publishing compiled wasm projects to the npm registry for the consumption
-of js devs using the npm CLI, yarn, or any other CLI tool that interfaces
-with the npm registry.
+this tool seeks to be a one-stop shop for building and working with rust-
+generated webassembly that you would like to interop with JavaScript, in the
```
Comment (mgattozzi) [inline README.md]: webassembly -> WebAssembly or wasm

File: README.md [hunk 6]
```diff
@@ -1,13 +1,17 @@
...
+browser or with Node.js. `wasm-pack` helps you build and publish rust-generated
+web assembly to the npm registry to be used alongside any other javascript
+package in workflows that you already use, such as [webpack] or [greenkeeper].
```
Comment (mgattozzi) [inline README.md]: Maybe mention other bundlers like Browserify (not sure about parcel), but it'll help show we're not partial to any one bundler

Comment (ashleygwilliams) [reply README.md]: do we know if those work yet? i'm super down to list them but i dont want to promise something that doesn't exist yet. we might want a "can i use?" at some point for bundlers

Comment (mgattozzi) [reply README.md]: Well if it's a JS package we're making my understanding is it just should? That was what I thought. Maybe @alexcrichton has a better understanding here. If not then we can expand it later.

Comment (ashleygwilliams) [reply README.md]: there are some bugs/blockers in other bundlers last i checked, but that may have changed!

Comment (mgattozzi) [reply README.md]: Cool well lets just table this comment for now. We can look it up later and update this as necessary

Comment (sendilkumarn) [reply README.md]: May be linking [here](https://github.com/rustwasm/team/blob/master/goals/bundler-integration.md#details) would help

File: README.md (hunk 3 above)
Comment (ashleygwilliams) [reply README.md]: why not the hyphen? the hyphen feels correct here... as rust modifies how it is generated?

**Comment by ashleygwilliams** [comment]
@mgattozzi pushed up almost all the changes you asked for :) lemme know if this is good to go and i'll fixup and merge

File: README.md [hunk 7]
```diff
@@ -1,13 +1,19 @@
...
+> your favorite rust -> wasm workflow tool!
 
 [![Build Status](https://travis-ci.org/ashleygwilliams/wasm-pack.svg?branch=master)](https://travis-ci.org/ashleygwilliams/wasm-pack)
 [![Build status](https://ci.appveyor.com/api/projects/status/7jjuo5wewu9lyyfi?svg=true)](https://ci.appveyor.com/project/ashleygwilliams/wasm-pack)
 
-the goal of this project is to create a portable command line tool
-for publishing compiled wasm projects to the npm registry for the consumption
-of js devs using the npm CLI, yarn, or any other CLI tool that interfaces
-with the npm registry.
+this tool seeks to be a one-stop shop for building and working with rust-
```
Comment (mgattozzi) [inline README.md]: this -> This

File: README.md [hunk 8]
```diff
@@ -1,13 +1,19 @@
...
+generated WebAssembly that you would like to interop with JavaScript, in the
+browser or with Node.js. `wasm-pack` helps you build and publish rust-generated
+WebAssembly to the npm registry to be used alongside any other javascript
+package in workflows that you already use, such as a bundler like 
+[webpack] or [greenkeeper]. for more details on bundler support, read [here][bundler-support].
```
Comment (mgattozzi) [inline README.md]: for -> For

**Pull request merged**